### ISSUE TITLE
NodeJS broken in latest update

### ISSUE THREAD

**Issue by davemackintosh** [termux/termux-packages#3612]
**Problem description**
I ran `pkg up` earlier today to get the latest and greatest of my installed packages and went to carry on working but it turns out that the nodeJS package is broken.

Any call to npm, or any package installed by npm results in an `invalid system call` error being thrown. 

I've tried 

* to downgrade but `apt-cache` shows no installable versions other than latest. 
* restarting my phone
* uninstalling node and reinstalling
* uninstalling Termux and reinstalling

**Steps to reproduce**
Install `nodejs` (should install 11.13.0) and run `npm`

**Expected behavior**
Any npm command to work but not even `npm version` works.

**Additional information**
```
 termux-info                      ✔  7  19:58:48
Updatable packages:
All packages up to date
Subscribed repositories:
https://dl.bintray.com/grimler/science-packages-21 science/stable
https://dl.bintray.com/grimler/game-packages-21 games/stable
https://termux.net stable/main
https://termux.net stable/main
System information:
Linux localhost 4.9.59-15367606 #1 SMP PREEMPT Wed Feb 20 22:25:51 KST 2019 aarch64 Android
Termux-packages arch:
aarch64
Android version:                                        9
Device manufacturer:                                    samsung                                                 Device model:
SM-G965F
```

Depending on problem, additional information may be requested:

1. Android warning/error log: `logcat -d "*:W"`.
2. Output of strace: `strace -fv -s 2048 -o strace.log {program name}`.
3. If program write it's own log files, you may need to attach them.

**Comment by davemackintosh** [comment]
Sorry, I did search but didn't find an existing issue.thabks for your hard work 🤘

**Comment by davemackintosh** [comment]
A note for anyone coming from Google. You need to run `pkg up` before updating your sources otherwise you'll get a GPG signature error.

**Comment by fornwall** [comment]
This has been fixed in version `11.13.0-1` of the nodejs package - run `pkg up` to update!
